src/domains/User/UserProfile/UserProfile.tsx

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
-import { Avatar } from '@material-ui/core';
+import { Avatar, Divider } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-// import DoneIcon from '@material-ui/icons/Done';
+import Button from '@material-ui/core/Button';
+import type { RegisterForm, User } from 'prytaneum-typings';
 
+import Form from 'components/Form';
 import TextField from 'components/TextField';
+import FormContent from 'components/FormContent';
+import PasswordResetForm from 'domains/Auth/PasswordResetForm';
+import API from 'domains/Auth/api';
+import useEndpoint from 'hooks/useEndpoint';
+import useSnack from 'hooks/useSnack';
+import useUser from 'hooks/useUser';
+import useForm from 'hooks/useForm';
 
 interface Props {
     // eslint-disable-next-line react/require-default-props
     img?: string;
+    id?: string
+    // safeUser?: ClientSafeUser;
+    // forceNoLogin?: boolean;
 }
```

Comment:
Clean up the props a bit and lets make default ones.  

```tsx
UserProfile.defaultProps = {
  img: undefined,
  id: undefined
```

Reply:
Done